### ISSUE TITLE
ESP32 support?

### ISSUE THREAD

**Issue by donovanmagryta** [martin-ger/esp_wifi_repeater#211]
Does this work with the ESP32 based board?

**Comment by martin-ger** [comment]
No, it has not yet been ported. However, as CPU speed is not a limiting factor, I don't think performance will be much better there - more memory might help to avoid buffer shortages...

**Comment by WayneKeenan** [comment]
see #24
